src/App.js

```diff
@@ -63,7 +63,8 @@ class App extends Component {
             .set({
               name: FBUser.displayName,
               userId: FBUser.uid,
-              likedRecipes: []
+              likedRecipes: [],
+              profileImageUrl: "https://i.imgur.com/XnEPZt3.jpg"
```

Comment:
Das Bild kann nach dem Profil Merge weg oder? Ansonsten hier meine Suggestion
```suggestion
              profileImageUrl: ""
```